### PR TITLE
feat: notify when client is disconnected from server

### LINKED ISSUE #1598: Let user know that Lan-Party is over/suspended
**What?**

Communicate to user that Athens client is disconnected from Lan-Party.

**Why?**

When we loose connection with Lan-Party player is unaware of it.
We should let the player know that Athens client is disconnected from Lan-Party.

**How?**

*Find 3 Solutions*

### PR DESCRIPTION
**Pull request by juniusfree**
closes #1598 

The summary of the decisions made about this issue can be found here https://github.com/athensresearch/athens/issues/1598#issuecomment-907512172

### CODE REVIEW

File: src/cljs/athens/views/pages/core.cljs [hunk 1]
```diff
@@ -51,6 +51,9 @@
 (defn view
   []
   (let [route-name (rf/subscribe [:current-route/name])]
+    ;; TODO: create a UI to inform the player of the connection status
+    (when (= @(rf/subscribe [:connection-status]) :reconnecting)
+      (rf/dispatch [:alert/js "Oops! Connection Lost. Reconnecting..."]))
```
Comment (neotyk) [inline src/cljs/athens/views/pages/core.cljs:56]: will this display alert every few seconds when we try to reconnect?

Comment (neotyk) [reply src/cljs/athens/views/pages/core.cljs:56]: It's good for now.

**Review by neotyk — APPROVED** [review]
well done

File: src/cljs/athens/views/pages/core.cljs (hunk 1 above)
Comment (juniusfree) [reply src/cljs/athens/views/pages/core.cljs:56]: > will this display alert every few seconds when we try to reconnect?

No, it's not.

**Pull request merged**